product-service/src/handlers/getProductFromDB.js

```diff
@@ -0,0 +1,31 @@
+import { Client } from 'pg';
+import { successResponse, errorResponse } from '../utils/api';
+import DB_CONFIG from '../db/config';
+import commonMiddleware from '../utils/middleware';
+
+async function getProduct(event) {
+  const client = new Client(DB_CONFIG);
```

Comment:
Would be great to use optimization here - https://docs.aws.amazon.com/lambda/latest/operatorguide/static-initialization.html